app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -124,7 +125,7 @@ open class DataStore(
         setupAutoLock()
     }
 
-    private fun deleteCredentials(item: ServerPassword?) {
+    private fun delete(item: ServerPassword?) {
```

Comment:
Nit: Is `item` ever `null`? Callers should work out if they want to delete _something_ rather than sending it to the `delete` method for it to check if it's `null`.